Format Standardization #1

TYWong99 opened the pull request:
Checking your opinion on the new standard for format, mostly regarding "comment simplication", "divider", "line spacing", and "including lines in CLI into main()".

Affected file:
 - tiled_tils.py
 - cli_collide.py
 - level_playdo.py

Also simplify the iteration method in cli_collide.py. I tested it and should be functional now, but I will wait until next pull request to replace with the new methods.

TYWong99 committed:
Format Standardization #1

Communicating on the new standard for format, mostly regarding comment simplication, divider, line spacing

Affected file:
 - tiled_tils.py
 - cli_collide.py
 - level_playdo.py

Also simplify the iteration method in cli_collide.py. Tested and currently functional, but will wait until next pull request to replace with the new methods.

qtran122 commented on logic/common/tiled_utils.py:
```diff
@@ -4,72 +4,70 @@
+    Takes a TILED layer encoded string and converts it into a tiles2d
+      encoded_str - string containing a layer's 'data' in TILED, zlib-encoded and base64-compressed
+      row_width   - integer indicating number of cells each row should contain
+      tiles2d     - 2d array of Tile IDs, first row is at top
```
I'm not a fan of this type of spacing - it creates unseemly gaps in the description IMO. Let's just use one space after the variable name

qtran122 commented on logic/common/tiled_utils.py:
```diff
@@ -4,72 +4,70 @@
-    # Convert the input array to a numpy array with type uint32
-    np_array = numpy.array(tiles2d, dtype=numpy.uint32)
+    Takes a TILED layer encoded string and converts it into a tiles2d
+      tiles2d     - 2d array of Tile IDs, first row is at top
```
I'm not a fan of this type of spacing - it creates unseemly gaps in the description IMO. Let's just use one space after the variable name

qtran122 commented on logic/common/tiled_utils.py:
```diff
@@ -4,72 +4,70 @@
-    >>> [[1, 2], [3, 4]]
+    '''
+    Removes empty rows and columns from a Tiles2D.
+      tiles2d     - 2d array of Tile IDs, cells with value 0 are "empty tile"
```
I'm not a fan of this type of spacing - it creates unseemly gaps in the description IMO. Let's just use one space after the variable name

qtran122 reviewed (changes requested):
Looks good! Just a few small comments about the indent for function params

TYWong99 committed:
Format Standardization #1b

Removed extra space character before hyphen in tiled_utils.py

Addition of 2 HowTo files

qtran122 commented on HowTo_Execute.txt:
```diff
@@ -0,0 +1,25 @@
+(WIP) User's Manual
```
Can you put all your HowTo's into a folder called "tutorials"?

Also, rename the tutorial text to use lowercase & be more descriptive : "how_to_pull_request" and "how_to_execute_cli"

qtran122 reviewed (changes requested):
Good job fixing the previous issues. The new files though I would like to be tidied up a bit. See comments

TYWong99 committed:
Format Standardization #1c

Moved how-to files to tutorials folder. Also renamed with snake case.

qtran122 commented:
LGTM
